### ISSUE TITLE
Something for the wiki

### ISSUE THREAD

**Issue by channeleaton** [Mixd/wp-deploy#71]
Thanks for this great setup! Something that has always bugged me with deploying w/ Capistrano is having to reload PHP after deployment. There are all kinds of hacks out there to run `service php5-fpm reload` but most pose a security risk. I did find [this fix for Nginx](http://www.sobstel.org/blog/php-opcache-refresh-symlinks-nginx-php-fpm/) that keeps you from having to do that. In your `location ~ \.php` block of the Nginx site config, just add the following two lines:

```
fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
fastcgi_param DOCUMENT_ROOT $realpath_root;
```

**Comment by lukewhitehouse** [comment]
Hi @channeleaton I'm not sure what your server setup is like but we've never had any problems on deployment and php. We're just running a very basic LAMP stack, whats your server like?

**Comment by channeleaton** [comment]
I'm running Ubuntu 14.04 and Nginx w/ PHP5-FPM. The issue I've encountered with this deployment method, as well as a Laravel-capistrano deployment, is the server 'sticks' to the real path of the current revision. When visiting the deployed site my changes never appeared. I had only been able to fix it by restarting php5-fpm manually or using some dirty hacks with capistrano.

Adding those two lines above to the site's server block in Nginx somehow prevents me from having to do that. Plugin & theme updates are available as soon as the deployment is complete.
